README.md

```diff
@@ -1,43 +1,81 @@
-# Secvisogram Backend
+# BSI Secvisogram CSAF Backend
 
 ![Coverage](https://raw.githubusercontent.com/secvisogram/csaf-cms-backend/badges/.github/badges/jacoco.svg)
 
+- [About The Project](#about-the-project)
+- [Getting started](#getting-started)
+- [How to use](#how-to-use)
+- [Developing](#developing)
+- [Contributing](#contributing)
+- [Dependencies](#dependencies)
+
+## About The Project
+
+TODO
+
+[(back to top)](#bsi-secvisogram-csaf-backend)
+
 ##  Getting started
 
-The configuration of the application as well as the compose file is done in 
+To run the CSAF CMS server you need the following:
+
+- [Keycloak](https://www.keycloak.org/)
+- A proxy like [oauth2-proxy](https://oauth2-proxy.github.io/oauth2-proxy/)
+- [CouchDB](https://couchdb.apache.org/)
+
+You can find an example setup for local development in the 'compose.yaml' and 
+an example configuration for Keycloak in 'keycloak/csaf-realm.json'. You 
+can take this as a starting point, but please check the documentation of the
+individual projects for a proper production setup.
+
+The [secvisogram](https://github.com/secvisogram/secvisogram) frontend is usable
+as a standalone version without this server. You can still use this standalone
+mode if the frontend is not behind the proxy, like in the development setup.
+In this case the login is only required to manage documents on the server 
+or validate against the [validator service](https://github.com/secvisogram/csaf-validator-service).
```

Comment:
Is this really suitable here describing how the frontend can be used with and without the backend? I actually find it kind of confusing what "In this case" means here.